jenkins/internal-ci/centos-7.9.2009/branch/motr.groovy

```diff
@@ -151,7 +151,7 @@ pipeline {
                                     string(name: 'branch', value: "${branch}")
                                 ]
                                 env.HARE_BUILD_NUMBER = hareBuild.number
-                            }catch (err){
+                            }catch (err) {
```

Comment:
Should we also change `}catch` to `} catch`

Reply:
This change` } catch` was not suggested.